frontend/simple/views/Invite.vue

```diff
@@ -164,21 +272,73 @@ export default {
             },
             mailbox
           )
-          await backend.publishLogEntry(member.state.attributes.mailbox, invite)
-
-          // We need to make a record of the invitation in the group's contract
-          const latest = await backend.latestHash(this.$store.state.currentGroupId)
-          const invited = new Events.HashableGroupRecordInvitation(
-            {
-              username: member.state.attributes.name,
-              inviteHash: invite.toHash(),
-              sentDate
-            },
-            latest
-          )
-          await backend.publishLogEntry(this.$store.state.currentGroupId, invited)
+          const state = await latestContractState(member.contractId)
```

Comment:
This appears to be unnecessarily re-fetching the entire history of the contract when that expensive operation has already been done on line 241.